catalogue/webapp/pages/concept.tsx

```diff
@@ -128,81 +109,75 @@ export const ConceptPage: NextPage<Props> = ({
-          <ConceptDescription>
-            <h1 className="font-intb">{conceptResponse.label}</h1>
-            {/* TODO dynamise */}
-            {FAKE_DATA.description && (
-              <p className="font-size-5">{FAKE_DATA.description}</p>
-            )}
-            {/* TODO dynamise */}
-            {FAKE_DATA.urls?.length > 0 &&
-              FAKE_DATA.urls.map(link => {
-                /* TODO Could they be internal links? Check if external to display arrow, decide on rel. */
-                return (
-                  <a
-                    key={link.url}
-                    href={link.url}
-                    rel="nofollow"
-                    className={font('intr', 6)}
-                  >
-                    {link.label} ↗
-                  </a>
-                );
-              })}
-          </ConceptDescription>
+          <TypeLabel>{conceptResponse.type}</TypeLabel>
+          <Space v={{ size: 's', properties: ['margin-top', 'margin-bottom'] }}>
+            {/* TODO only getting Subjects, confirm if it will change or not */}
+            <HeroTitle>{conceptResponse.label}</HeroTitle>
+            {/* TODO get copy from Jonathan */}
+            <ConceptDescription className="font-size-5">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam
+              dapibus suscipit enim nec aliquam.
+            </ConceptDescription>
+          </Space>
         </div>
       </ConceptHero>
 
+      {/* Images */}
       {hasImages && (
-        <ConceptImages
-          as="section"
-          v={{ size: 'xl', properties: ['padding-top', 'padding-bottom'] }}
-        >
+        <ConceptImages as="section">
           <div className="container">
-            <h2 className={`sectionTitle ${font('wb', 3)}`}>Images</h2>
+            <h2 className="h2 sectionTitle">Images</h2>
 
             {hasImagesTabs && (
               <TabNavV2
+                id="images"
+                selectedTab={selectedImagesTab}
                 items={[
                   {
                     id: 'images-about',
-                    text: `Images about ${conceptResponse.label} ${
-                      imagesAbout ? `(${imagesAbout.totalResults})` : ''
-                    }`,
+                    text: (
+                      <>
+                        {`About this ${conceptResponse.type.toLowerCase()} `}
+                        <span className="is-hidden-s">{`(${imagesAbout.totalResults})`}</span>
+                      </>
+                    ),
                     selected: selectedImagesTab === 'images-about',
                   },
                   {
                     id: 'images-by',
-                    text: `Images by ${conceptResponse.label} ${
-                      imagesBy ? `(${imagesBy.totalResults})` : ''
-                    }`,
+                    text: (
+                      <>
+                        {`By this ${conceptResponse.type.toLowerCase()} `}
```

Comment:
is there a list of `conceptResonse.type`s? I'm wondering if this will always make sense

Reply:
I think this is probably fine with Meeting; even if not it shouldn't block merging this PR.

Reply:
Actually had to change it to

```
type ConceptType =
  | 'Subject'
  | 'Meeting'
  | 'Organisation'
  | 'Person'
  | 'Concept'
  | 'Period'
  | 'Place';
```

As the last 3 are used in the Works API as well.